src/app/garfile/arrival/post.controller.js

```diff
@@ -86,7 +86,7 @@ const buildValidations = (voyage) => {
   const validations = [
     [new ValidationRule(validator.realDate, 'arrivalDate', arriveDateObj, realDateMsg)],
     [new ValidationRule(validator.currentOrFutureDate, 'arrivalDate', arriveDateObj, futureDateMsg)],
-    [new ValidationRule(validator.dateNotTooFarInFuture, 'arrivalDate', arriveDateObj, __('field_arrival_date_validation'))],
+    [new ValidationRule(validator.dateNotTooFarInFuture, 'arrivalDate', arriveDateObj, 'Arrival date cannot be more than 1 month in the future')],
```

Comment:
Should we add the 'must be today' part here as well ?

Reply:
must be today is handled in a separate message:
`const futureDateMsg = 'Arrival date must be today or in the future';`